spa/src/utils/dynamicTableHelper.tsx

```diff
@@ -0,0 +1,149 @@
+import Avatar from "@atlaskit/avatar";
+import styled from "@emotion/styled";
+import Badge from "@atlaskit/badge";
+import { token } from "@atlaskit/tokens";
+import Lozenge from "@atlaskit/lozenge";
+import { SuccessfulConnection } from "../rest-interfaces";
+import { ThemeAppearance } from "@atlaskit/lozenge/dist/types/Lozenge";
+
+type Row = {
+	key: string;
+	isHighlighted: boolean;
+	cells: { key: string | number; content: React.JSX.Element | string | number }[];
+};
+
+// eslint-disable-next-line react-refresh/only-export-components
+const RowWrapper = styled.div`
+	display: flex;
+	align-items: center;
+`;
+// eslint-disable-next-line react-refresh/only-export-components
+const AvatarWrapper = styled.span`
+	margin-right: ${token("space.200")};
+`;
+
+const ifAllReposSynced = (
+	numberOfSyncedRepos: number,
+	totalNumberOfRepos: number
+): number | string =>
+	numberOfSyncedRepos === totalNumberOfRepos ? totalNumberOfRepos :
```

Comment:
This would be a bit easier to read if it was structured like this:
```
numberOfSyncedRepos === totalNumberOfRepos 
  ? totalNumberOfRepos 
  : (totalNumberOfRepos ? `${numberOfSyncedRepos} / ${totalNumberOfRepos}` : "");
```
I don't think having a nested ternary is easy to read though. It might be worth having an `if` statement for clarity.

In regards to this specific ternary, I think there might be a better way to code this overall. Can we chat about this via zoom?

Reply:
The indentation here is a code convention related thing, so will fix it in a separate PR. It will be done in [ARC-2467](https://softwareteams.atlassian.net/browse/ARC-2467)

Reply:
> In regards to this specific ternary, I think there might be a better way to code this overall. Can we chat about this via zoom?

I think it's worth reviewing the implementation of this code, aside from the implementation

Reply:
Changed the ternary to a if else for better readability.